include/wx/gtk/dvrenderer.h

```diff
@@ -93,6 +97,9 @@ class WXDLLIMPEXP_ADV wxDataViewRenderer: public wxDataViewRendererBase
     // doing this
     bool m_usingDefaultAttrs;
 
+    // the item currently being rendered
+    wxDataViewItem m_currentItem;
```

Comment:
I think this should be called `m_itemBeingRendered` to avoid/reduce confusion with `wxDataViewRendererBase::m_item` (which could be renamed to `m_itemBeingEdited`, with shorter names being preferable for both of them if you can think of something not too ungrammatical).